tests/tests_geomstats/test_riemannian_kmeans.py

```diff
@@ -83,3 +83,23 @@ def test_hypersphere_kmeans_initialization(self):
         result = centroids.shape
         expected = (n_clusters, 3)
         self.assertAllClose(expected, result)
+
+    def test_hypersphere_kmeans_init_array(self):
+        gs.random.seed(55)
+
+        n_features = 3
+        n_clusters = 3
```

Comment:
Use a different number of clusters and features, otherwise the shape is `3x3`, which tells very little about the correctness of the result.

Reply:
Sure. I also added a helper function to make it less redundant.